infra/database/type/mysql/src/main/java/org/apache/shardingsphere/infra/database/mysql/metadata/data/loader/MySQLMetaDataLoader.java

```diff
@@ -62,21 +62,41 @@ public final class MySQLMetaDataLoader implements DialectMetaDataLoader {
     private static final String CONSTRAINT_META_DATA_SQL = "SELECT CONSTRAINT_NAME, TABLE_NAME, REFERENCED_TABLE_NAME FROM information_schema.KEY_COLUMN_USAGE "
             + "WHERE TABLE_NAME IN (%s) AND REFERENCED_TABLE_SCHEMA IS NOT NULL";
     
+    private static final String TABLE_TYPE_META_DATA_SQL = "SHOW FULL TABLES";
```

Comment:
@Pace2Car Hi, can we get table type from information_schema.tables?
> SELECT TABLE_NAME, TABLE_TYPE FROM information_schema.tables WHERE TABLE_NAME IN ('TABLES');

Reply:
Refactored, THX